src/components/CommentsContainer.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+import { fetchReviewComments } from "../api"
+import { CommentCard } from "./CommentCard"
+import "../styles/CommentContainer.css"
+
+export const CommentsContainer = ({review_id, user}) => {
+
+    const [comments, setComments] = useState([])
+
+    useEffect(() => {
+        fetchReviewComments(review_id)
+        .then((retrievedComments) => {
+            setComments(retrievedComments)
+        })
+    }, [])
+
+    return(
+        <div className="comments-container">
+            {comments.map((comment) => {
```

Comment:
Nice, but consider what will happen if a review has no associated comments yet. You currently aren't going to display anything and a user may assume the comments just haven't loaded correctly. Add in an alternate message and see if you can render it conditionally depending on the length of retrievedComments.